### PR TITLE
Add an $(XML2RFCV3) variable to opt into the xml2rfc v3 path.

### PR DESCRIPTION
**Pull request by jyasskin**
The other tools aren't totally ready for this, but this'll make it easier to find their bugs.

### CODE REVIEW

File: main.mk [hunk 1]
```diff
@@ -55,6 +55,8 @@ endif
 %.xml: %.org
 	$(oxtradoc) -m outline-to-xml -n "$@" $< > $@
 
+XML2RFCV3OPTION := $(if $(V3),--v3)
```
Comment (martinthomson) [inline main.mk]: I've tended to use the following form:

```suggestion
ifeq (true,$(XML2RFCV3))
XML2RFCV3 := --v3
endif
```

It's more verbose, but it makes the check - and the variable - more explicit.

File: main.mk [hunk 2]
```diff
@@ -55,7 +55,11 @@ endif
 %.xml: %.org
 	$(oxtradoc) -m outline-to-xml -n "$@" $< > $@
 
-XML2RFCV3OPTION := $(if $(V3),--v3)
+ifeq (true,$(XML2RFCV3))
+XML2RFCV3 := --v3
```
Comment (martinthomson) [inline main.mk]: This is a little odd, overwriting the value of the tested variable, but it does work.

Comment (jyasskin) [reply main.mk]: Oh, I thought that's what you'd suggested in https://github.com/martinthomson/i-d-template/pull/195#discussion_r332904139. I don't have any particular opinion about the best way to do this. :)

Comment (martinthomson) [reply main.mk]: I must have typed the same thing twice :/  I'll sort it out.

**Comment by jyasskin** [comment]
Thanks! (8578e0572a00625d4972d3d1cc4ce3081dc50f4a)

**Pull request closed without merging**